### ISSUE TITLE
Why is ticketing in there?

### ISSUE THREAD

**Issue by johnnonolan** [opentable/design-tokens#211]
Everything else in this repo is granular why do we have this? Feels like the antithesis to what design tokens is.
https://github.com/opentable/design-tokens/tree/master/OTKit/otkit-ticketing-desktop

**Comment by lastquestion** [comment]
It was merged in this PR #168. I didn't agree at the time and I don't agree with it now. 
After some discussion with @karuto a few weeks / months ago I think we were aligned with getting rid of it from this repo. That's what I think we should do.

@acolchado also pointed this out today, as well.

**Comment by matteofigus** [comment]
@johnnonolan I think the goal of this repo is, in my view, to allow progress in a super dynamic world which is designing interfaces. In this perspective, I think the goal around a "functional token" was to try and learn - because why not.

I think that everything that is here should be here if it makes sense from a design point of view. This means that if we think as engineers that it doesn't, possibly we should talk 👍 

@francesyun and @mannionaco what are your thoughts?

Personally I feel theoretically in line with @johnnonolan and @lastquestion but in practice I don't mind. :)

**Comment by lastquestion** [comment]
@karuto , I've opened this PR #216 to delete it. Let me know your thoughts!

**Comment by jrolfs** [comment]
I don't think this belongs here, but that's not to say it's existence in this format is altogether bad design.

<p align="center"><em>* steps onto soap-box *</em></p>

Perhaps this belongs in its own *"token"* module? I've been thinking about how larger pieces of UI/style can be shared effectively and there's pretty clearly a substantial gap between **design-tokens** and **ot-react-ui-components**.

Adding another layer of transitive dependency when we haven't yet proven our ability to keep things up to date with the current system (that banks heavily on **semver** versioning *as-primitive-and-or-granular-as-possible* dependencies) is definitely risky. Standardization on **React** as a definition for **"component"** helps close the gap between **[token]** ⫸ **[component]**, but fails to acknowledge our state of transition and need for migration paths and interop.

I don't think we'll figure any of this out without experimentation, and perhaps we can just deal with legacy stuff (that can't take advantage of direct `(React(design-token))` composition) on a case-by-case basis knowing we're planning on ditching those implementations.

Because, this:

```sh
[token] ⫸ [‹composite› token] ⫸ [(react) component]
```

seems potentially a little unwieldy, especially when this:


```sh
[token] ⫸ [‹composite› token] ⫸ [(react) component] ⫸ [‹composite› component]
                                  # do we attempt to share this ^ ^ ^ ^ ^ ?
```

Hasn't even really been worked out〈(°。°)

<p align="center"><em>* steps down from soap-box, the few people that showed interest nowhere in sight... 🦗🦗🦗🦗🦗 *</em></p>

*Srsly though – if anyone made it this far (and this made any sense at all) I'm definitely curious to hear your thoughts.*

Even thought it won't be nearly abstract enough \*cough\*, I think I'm okay with leaning in on React and encoding composite design ideas into React components. Even with making that decision/assumption there are plenty of necessary abstractions – which I think we need to focus on defining :)

**TLDR;** `(︶｡︶) ... (︶⁠〇︶) ....... (° ⌓⁠° )`

⋮
⋮
⋮ 

@lastquestion @karuto @johnnonolan @francesyun (don't feel obligated to 📖👀)

**Comment by karuto** [comment]
@johnnonolan @lastquestion I was not aware of this whole situation. I think this is just the symptom of a deeper problem: that we failed to agree on what design tokens' role is within our design / engineering ecosystem.

If you treat design tokens as the repository to only store the most granular, universal atomic level design decisions - say, "everything in design tokens should be applicable to every single consumer micro site" - then sure, I agree with you, you can remove it.

Here is my take: design tokens should encompass all sharable design decisions on consumer platform. Not just the most granular and absolutely universal ones; if a design decision is of shared nature, it should have a representation in design tokens. In the case of ticketing, this is a project that spans across multiple teams and microsites but is governed by some shared design decisions.

If you don't think design tokens is the right place to put it, it would be great to share your insight on your alternative proposal to this problem. Deleting a token because you don't think it "belongs", without addressing the core of the problem or discussing alternate solutions, I don't think this will be a constructive thread.

@jrolfs I'm sorry man, I don't think I understand everything you said. Are you suggesting we should bake these design decisions into React components? In the context of this problem, the ticketing front end pieces live in multiple projects with different frameworks, and we don't have a central shared React component to encode our design decision into. Let me know what you think!

**Comment by francesyun** [comment]
This thread is too long, so I didn't read everything.

My 2 cents:
I would rather the tokens be under a "modal" family instead of a "ticketing" family. Ticketing does feel too granular. But the tokens themselves are useful because modals are a component that should exist across many areas of the website.

**Comment by karuto** [comment]
@francesyun thanks for replying. I'm ok with some elements of it being in `otkit-modals` or something like that. But I have started to doubt whether there is an appetite on design / engineering for things like a product-based decision sharing, something like ticketing that is beyond the "atomic" level of UI components. I thought there was an appetite for that, based on my experience working with @matteofigus and @mannionaco, but that reality seems shaky judging by the overwhelmingly negative feedback from this thread. If there is no need to share design decisions like these, then I'm ok to remove it.

**Comment by lastquestion** [comment]
@karuto - I think that we ought to prefer encoding such design decisions into react components first. If it turns out that the design decisions need to exist across multiple tech stacks we can then decide what to do - there are many options - e.g. whether copy and paste, sharing via NPM package, etc. 

To bring it back to the concrete topic - I feel that we could implement modals in react and then encode these design decision values in the styling of that component directly.
